### PR TITLE
fix(fxa-57): add metrics success

### PR DESCRIPTION
**Pull request by vladikoff**
Closes https://github.com/mozilla/fxa-auth-server/issues/1082

### CODE REVIEW

**Comment by vladikoff** [comment]
@rfk @billmaggs r?

Talked to @ryanfeeley about this.

File: features/FxA-57-verify-reminder/README.md [hunk 1]
```diff
@@ -12,7 +12,17 @@ As a new Firefox Accounts user,
...
  
  The second email arrives after 168 hours since sign up.
- 
+
+## Success criteria
+
+We are planning to get the results from the following metrics:
+
+- Measure number of people that verify using the first email and second email.
+We would use the total number of emails sent vs verified.
```
Comment (rfk) [inline features/FxA-57-verify-reminder/README.md]: Channeling @ckarlof, let's be very explicit that we'll measure it "as a percentage of number of emails sent".

File: features/FxA-57-verify-reminder/README.md [hunk 2]
```diff
@@ -12,7 +12,17 @@ As a new Firefox Accounts user,
...
+
+- The ultimate measure of success for this feature is if the user connects to sync.
+We will use the retention metrics pipeline to measure "account.signed" events that happened after "verification reminder" events
```
Comment (rfk) [inline features/FxA-57-verify-reminder/README.md]: Does the "verification reminder" event indicate that a reminder was sent, or that it was acted on?

We could be more explicit here as well, with something like "the percentage of 'verification reminder sent' events that are followed by an 'account.signed' event up to X days later".

**Comment by rfk** [comment]
I left some nits about being more precise about what we'll measure, but I'm otherwise r+ as I think this nicely captures what it would mean for the feature to succeed.  In particular, if there's a big discrepancy between the two measures and people are verifying but not subsequently connecting to sync, then we've got follow-up work to do.

**Comment by vladikoff** [comment]
Updated, thank you for review Ryan!

File: features/FxA-57-verify-reminder/README.md (hunk 1 above)
Comment (ckarlof) [inline features/FxA-57-verify-reminder/README.md]: +1 in defining these as relative values

**Pull request merged**